doc/solvers.md

```diff
@@ -2,98 +2,110 @@
+The goal is then to find $x$ such that:
 
 ```{math}
 F(x) = 0
 ```
 
-As finding an exact solution of a nonlinear system is very difficult, we actually try to find $x$ such as:
+Computationally, this translates to finding a solution $x$ such that:
 ```{math}
-||F(x)||_{\infty} < \epsilon
+||F(x)||_{\infty} < \varepsilon
 ```
-With $\epsilon$ being the parameter `tolerance` that the user can choose (by default `1e-6`).
+With $\varepsilon$ being a small *tolerance*.
+In code, $\varepsilon$ can be set with `en.solve_load_flow(tolerance=...)` (by default `1e-6`).
+
+There are several algorithms to solve this kind of problems. In Roseau Load Flow, the following
+algorithms are available:
 
 
 ## Newton-Raphson
 
-This is the classical Newton-Raphson solver.
+This is the classical [*Newton-Raphson* method](https://en.wikipedia.org/wiki/Newton%27s_method).
+
+First, an initial solution $x_0$ is chosen by initializing the voltages either by propagating the
+voltage of the sources or by re-using the results from the last successful run. The choice of
+either option depends on the `warm_start` argument to the `en.solve_load_flow()` method.
 
-We start by choosing an initial solution $x_0$ by propagating the voltages, or re-using the previous results if a load
-flow has already been run (this behaviour actually depends on the `warm_start` parameter which default to `True`).
 Then, multiple iterations are made with:
 
 ```{math}
 :label: step
 x_{n+1} = x_n - J_F^{-1}(x_n)F(x_n)
 ```
 
-with $J_F$ the jacobian of $F$.
+with $J_F$ being the jacobian of $F$.
 
-The iterations are stopped when we find $x_n$ such as $||F(x_n)||_{\infty} < \epsilon$, or when the number of iterations
-has exceeded the parameter `max_iterations` (in which case we consider that the algorithm can't converge).
+The algorithm stops when it finds a solution $x_n$ such that $||F(x_n)||_{\infty} < \varepsilon$
+within a maximum number of iterations (modify with `en.solve_load_flow(max_iterations=...)`). If
+the maximum number of iterations is exceeded, the algorithm did not converge and the execution
+fails.
 
 ### Parameters
 
-The user can specify one parameter:
-- `"linear_solver"`: the linear solver used to compute $J_F^{-1}(x)F(x)$. Currently only `"SparseLU"` is available.
+The *Newton-Raphson* algorithm accepts one parameter:
+- `"linear_solver"`: the linear solver used to compute $J_F^{-1}(x)F(x)$. Currently only the
+  `"SparseLU"` option is available.
 
 ## Goldstein and Price
 
-This is a variant of the classical Newton-Raphson algorithm with a linear search.
+This is a variant of the classical *Newton-Raphson* algorithm with a linear search.
 
-At each iteration we're choosing $x_{n+1}$ such as
+At each iteration, $x_{n+1}$ is calculated using:
 ```{math}
 :label: linear_search_step
-x_{n+1} = x_n + td(x_n)
+x_{n+1} = x_n + t d(x_n)
 ```
 with $d = -J_F^{-1}F$
 
-For the classical Newton-Raphson algorithm, $t=1$ is chosen for the next iterate. The idea of the linear searches
```

Comment:
I am not sure what this line means.

Reply:
It just means that in the "original" Newton-Raphson algorithm, the next iterate is `x_{n+1} = x_n + d(x_n)`.